### DISCUSSION TITLE
Show server localized time in status bar

### DISCUSSION THREAD

**Discussion by christoferd** [opcodesio/log-viewer#325] [Ideas]
Hi, a cool feature would be to show server localized time in status bar. 

I work on sites in other time-zones, so it would be handy to understand if a log entry happened recently.

![image](https://github.com/opcodesio/log-viewer/assets/1835883/e6c29291-3f72-40a0-ba71-6459a54a0ede)

**Comment by arukompas** [comment]
hey @christoferd , what about setting the timezone to your preferred one in the `config/log-viewer.php` ? See the `"timezone"` property.

You could set it to your local timezone, and then any datetimes displayed in the UI will be in that timezone :)
